### PR TITLE
message transformation examples

### PR DESCRIPTION
**Pull request by nilminiwso2**
Summary of change:

The same use case (SOAP backend and JSON client) is used for both of the following examples:
- JSON to SOAP conversion (JSON client sends a message to the SOAP backend)
- SOAP to JSON conversion  (SOAP backend responds to the JSON client)

Changed the synapse code accordingly. Tested.

### CODE REVIEW

File: en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+               <arg evaluator="json" expression="$.placeOrder.order.price"/>
+               <arg evaluator="json" expression="$.placeOrder.order.quantity"/>
+            </args>
+         </payloadFactory>
+         <log level="full"/>
+         <send>
+              <endpoint>
+                  <address uri="http://localhost:9000/services/SimpleStockQuoteService"/>
+              </endpoint>
+          </send>
```
Comment (sdkottegoda) [inline en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md]: The soapUI/Postman client will hang if we don't reply back

File: en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            <log level="full"/>
+            <xslt key="in_transform"/>
+            <property name="messageType" scope="axis2" type="STRING" value="text/xml"/>
+            <header name="Action" scope="default" value="urn:getQuote"/>
+            <enrich>
+                <source clone="true" xmlns:m0="http://services.samples" xpath="//m0:getQuote"/>
+                <target type="body"/>
+            </enrich>
+            <log level="full"/>
+            <send>
```
Comment (sdkottegoda) [inline en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md]: same as above

File: en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md (hunk 1 above)
Comment (nilminiwso2) [reply en/micro-integrator/docs/use-cases/examples/message_transformation_examples/json-to-soap-conversion.md]: resolved

**Comment by nilminiwso2** [comment]
Created new PR with suggested changes.
- https://github.com/wso2/docs-ei/pull/1108

**Pull request closed without merging**